api/public/schema/projectSchema.js

```diff
@@ -38,6 +38,10 @@ const schema = {
       type: "string",
       required: false
     },
+    commentIds: {
```

Comment:
Also, perhaps this should be an array so that multiple comments could be linked to a project

Reply:
So actually since Brandon is working on user comments he had a different setup for it so we don't need comment fields in the project or user schemas anymore